presto-orc/src/main/java/io/prestosql/orc/StripeReader.java

```diff
@@ -382,16 +386,22 @@ static boolean isIndexStream(Stream stream)
     private Map<Integer, List<BloomFilter>> readBloomFilterIndexes(Map<StreamId, Stream> streams, Map<StreamId, OrcChunkLoader> streamsData)
             throws IOException
     {
-        ImmutableMap.Builder<Integer, List<BloomFilter>> bloomFilters = ImmutableMap.builder();
+        HashMap<Integer, List<BloomFilter>> bloomFilters = new HashMap<>();
```

Comment:
Use `Map` for the declaration

Reply:
btw. this looks like `Multimap<Integer, BloomFilter>`